core/server/api/db.js

```diff
@@ -120,21 +126,21 @@ db = {
          *   - `onDestroyed` or `onDestroying` can contain custom logic
          */
         function deleteContent() {
-            return models.Base.transaction(function (transacting) {
+            return models.Base.transaction((transacting) => {
                 queryOpts.transacting = transacting;
 
                 return models.Post.findAll(queryOpts)
                     .then((response) => {
                         return Promise.map(response.models, (post) => {
-                            return models.Post.destroy(_.merge({id: post.id}, queryOpts));
+                            return models.Post.destroy(merge({id: post.id}, queryOpts));
```

Comment:
//cc @kirrg001 can you use `Object.assign` here?

Reply:
@vikaspotluri123 Thanks. we can do this. I am updating the PR